packages/lit-dev-server/tsconfig.json

```diff
@@ -1,13 +1,13 @@
 {
   "compilerOptions": {
     "incremental": true,
-    "tsBuildInfoFile": "./tsconfig.tsbuildinfo",
+    "tsBuildInfoFile": "./lib/.tsbuildinfo",
```

Comment:
Tiny nit: Other packages kept this as `"./lib/tsconfig.tsbuildinfo"`.

Reply:
Hm interesting. The default is actually `<outdir>/.tsbuildinfo` which matches this path (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html). So we could also just omit it.

Reply:
I might go through and standardize these in a followup.